python/paddle/fluid/layers/nn.py

```diff
@@ -3978,3 +3979,53 @@ def _is_list_or_turple_(data):
         attrs={"out_h": out_h,
                "out_w": out_w})
     return out
+
+
+def gather(input, index):
+    """
+    **Gather Operator**
```

Comment:
In Python API, there should not be any operator.

Reply:
Thx. Fixed.